### ISSUE TITLE
Redefinition of layout_remove_any() (first defined in xtensor/xeval.hpp)

### ISSUE THREAD

**Issue by krivenko** [xtensor-stack/xtensor-blas#178]
Hi there,

Since recently, my code has been failing to compile with the following error message.
```
/usr/include/xtensor-blas/xblas_utils.hpp:40:31: error: redefinition of ‘constexpr xt::layout_type xt::detail::layout_remove_any(xt::layout_type)’

         constexpr layout_type layout_remove_any(const layout_type layout)

...

/usr/include/xtensor/xeval.hpp:57:31: note: ‘constexpr xt::layout_type xt::detail::layout_remove_any(xt::layout_type)’ previously defined here

         constexpr layout_type layout_remove_any(const layout_type layout)
```

I believe the problem is caused by the recently merged  xtensor-stack/xtensor#2261 (commit https://github.com/xtensor-stack/xtensor/commit/40bf942fa4c33749fab915bb31c36b22d94830d9).

**Comment by JohanMabille** [comment]
Hi,

Thanks for reporting! I guess you're using xtensor master? I've pushed https://github.com/xtensor-stack/xtensor/pull/2275 to move `remove_layout_any` to the public API. Once `xtensor` has been released, I will rebase xtensor-blas and remove the duplicated definition.

**Comment by krivenko** [comment]
Thank you, everything compiles now!
Feel free to close this issue now or when the duplicated definition is removed from `xtensor-blas`.

**Comment by JohanMabille** [comment]
Fixed in #179
